packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -1,30 +1,14 @@
 import React, { createContext, useContext } from 'react'
-import { SynapseClientError } from '../../utils'
-import {
-  AsynchronousJobStatus,
-  ColumnModel,
-  QueryBundleRequest,
-  QueryResultBundle,
-} from '@sage-bionetworks/synapse-types'
+import { QueryBundleRequest } from '@sage-bionetworks/synapse-types'
 import { ImmutableTableQueryResult } from '../../utils/hooks/useImmutableTableQuery/useImmutableTableQuery'
 import { ReadonlyDeep } from 'type-fest'
+import { TableQueryUseQueryOptions } from '../QueryWrapper/TableQueryUseQueryOptions'
+import { LockedColumn } from '../../utils'
 
-type OptionalQueryBundleRequestFields = keyof Pick<
-  QueryResultBundle,
-  | 'queryResult'
-  | 'queryCount'
-  | 'selectColumns'
-  | 'maxRowsPerPage'
-  | 'columnModels'
-  | 'facets'
-  | 'sumFileSizes'
-  | 'combinedSql'
->
-
-export type QueryContextType<
-  TIncludedFields extends OptionalQueryBundleRequestFields = never,
-  ExcludeOtherFields extends true | false = false,
```

Comment:
Removed these generics which didn't really solve any problems, since it's impossible to know which fields are present in the QueryResultBundle until runtime